### PR TITLE
[Commerce] Creative Amends

### PR DESCRIPTION
**Pull request by gurmukhp**

### CODE REVIEW

File: css/templates/e-commerce/page.css [hunk 1]
```diff
@@ -1,7 +1,8 @@
+@import '../../core';
 @import 'basscss-responsive-type-scale';
 @import 'basscss-responsive-margin';
 @import 'basscss-responsive-padding';
-@import '../../core';
+@import '../../ampstart-base/bass-override-vars';
```
Comment (gurmukhp) [inline css/templates/e-commerce/page.css]: Re-adding base vars here as certain classes such as md-mt4 weren't correct.

File: css/templates/e-commerce/page.css [hunk 2]
```diff
@@ -189,26 +190,9 @@ amp-selector [option][selected] {
...
-
-.amp-carousel-button-prev {
-  left: 0;
-}
-
+.amp-carousel-button-prev,
 .amp-carousel-button-next {
-  right: 0;
-  transform: translateY(-50%) rotate(180deg);
+  display: none;
```
Comment (gurmukhp) [inline css/templates/e-commerce/page.css]: Hide arrows for all carousels except for related products

File: templates/e-commerce/product-listing.amp.html [hunk 3]
```diff
@@ -44,13 +44,15 @@ <h2 class="h5 mb2">Categories</h2>
...
-      <div class="md-commerce-header relative md-flex flex-wrap items-center mx1 md-mx0 md-mb2">
-        <h1 class="h3 mb1 mt2 md-mt2 md-mb2 md-ml0 flex-auto">Products</h1>
-        <div class="inline-block mr1 mb3 md-mb0">
-          {{#categories}} {{> partials/select.snip.html}} {{/categories}}
+    <div class="col-12 md-col-7 pt2 pb3 md-px4 md-pt1 md-pb7">
+      <div class="md-commerce-header relative md-flex flex-wrap items-center md-mx0 md-mb2">
+        <h1 class="h3 mb2 md-mt2 md-mb2 md-ml0 flex-auto px2">Products</h1>
+        <div class="commerce-listing-filters pt2 pb2 mb3 md-mb0">
+          <div class="inline-block mr1 pl2">
+            {{#categories}} {{> partials/select.snip.html}} {{/categories}}
```
Comment (camelburrito) [inline templates/e-commerce/product-listing.amp.html]: nit: cant you move the spacing classes to the select and get rid of the wrapper?

File: css/templates/e-commerce/page.css [hunk 4]
```diff
@@ -717,6 +728,33 @@ amp-carousel {
   border-top: 1px solid var(--color-separator);
 }
 
+.commerce-related-products amp-carousel {
+  padding-left: 40px;
+  padding-right: 40px;
+}
+
+.commerce-related-products .amp-carousel-button {
```
Comment (camelburrito) [inline css/templates/e-commerce/page.css]: if this applies to all your carousel buttons , its cool to do `.amp-carousel-button`

File: css/templates/e-commerce/page.css [hunk 5]
```diff
@@ -717,6 +728,33 @@ amp-carousel {
...
+  background: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="30px" height="30px" viewBox="0 0 24 24" fill="#000000"><path d="M15.41 16.09l-4.58-4.59 4.58-4.59L14 5.5l-6 6 6 6z"/><path d="M0-.5h24v24H0z" fill="none"/></svg>') 30px 30px transparent;
+  cursor: pointer;
+  height: 36px;
+  width: 36px;
+  top: 40%;
+}
+
+.commerce-related-products .amp-carousel-button-prev,
```
Comment (camelburrito) [inline css/templates/e-commerce/page.css]: ditto

Comment (gurmukhp) [inline css/templates/e-commerce/page.css]: Same as above.

File: templates/e-commerce/product-listing.amp.html (hunk 3 above)
Comment (gurmukhp) [inline templates/e-commerce/product-listing.amp.html]: Good idea, moved to partial and added classes as per design doc.

**Comment by gurmukhp** [comment]
Nits addressed and also renamed select in json to commerce-select to prevent `inputs.spin.html` from rendering selects.

**Comment by gurmukhp** [comment]
Thanks! Feel free to merge whenever ready.

File: css/templates/e-commerce/page.css [hunk 6]
```diff
@@ -140,6 +140,10 @@ dd:after {
   margin-bottom: var(--space-1);
 }
 
+.ampstart-input {
+  width: 100%;
```
Comment (gurmukhp) [inline css/templates/e-commerce/page.css]: Ensure input fields are full width on mobile. On desktop they are as wide as the column they're in.

**Pull request merged**